packages/remediations/package.json

```diff
@@ -7,7 +7,10 @@
         "access": "public"
     },
     "scripts": {
-        "build": "NODE_ENV=production webpack --config config/webpack.config.js",
+        "build": "NODE_ENV=production webpack --config config/webpack.config.js && npm run build:js && npm run build:esm && npm run build:umd",
```

Comment:
Is the webpack build still relevant?

Reply:
Yup, we still want to build the remediations wizard that is used in chrome. Now there's webpack build just for the wizard and rollup build for button.